### DISCUSSION TITLE
How do we change the keyboard shortcut to ungrab th ekeyboard/mouse?

### DISCUSSION THREAD

**Discussion by imthenachoman** [quickemu-project/quickemu#1371] [Quickemu]
My keyboard does not have an F12 key. How can I change the shortcut from `Shift + F12` to something else?

**Comment by Archie-2021** [comment]
I have the same problem. where can I configure this?

**Reply by philclifford** [reply]
It depends...If you are using `virt-viewer` as your spice client then `man virt-viewer` will show you that you have command line options to control such things:
``` text
      -H HOTKEYS, --hotkeys HOTKEYS
           Set global hotkey bindings. By default, keyboard shortcuts only work when the guest display widget does not have focus.  Any actions specified in
           HOTKEYS will be effective even when the guest display widget has input focus. The format for HOTKEYS is
           <action1>=<key1>[+<key2>][,<action2>=<key3>[+<key4>]].  Key-names are case-insensitive. Valid actions are: toggle-fullscreen, release-cursor, zoom-in,
           zoom-out, zoom-reset, secure-attention, usb-device-reset, smartcard-insert and smartcard-remove.  The "secure-attention" action sends a secure attention
           sequence (Ctrl+Alt+Del) to the guest. Examples:
             --hotkeys=toggle-fullscreen=shift+f11,release-cursor=shift+f12

             --hotkeys=release-cursor=ctrl+alt

```

If you install spice-vdagent you should find that you can just move out of the window (unless you are full-screen).  Depending on what display setting, viewer/client, host OS, guest OS, host display manager, ... YMMV and how you may control the keyboard shortcuts might vary.  You might be able to configure a keyboard mapping for F11-12 in your host session.
